pkg/generate/app/componentresolvers.go

```diff
@@ -197,21 +154,18 @@ type HighestUniqueScoreResolver struct {
 // Resolve resolves as the highest scored match returned by the Searcher, and
 // guarantees the match is unique (the only match with that given score)
 func (r HighestUniqueScoreResolver) Resolve(value string) (*ComponentMatch, error) {
```

Comment:
Maybe not, but I don't want to introduce even more churn in this PR.  They're in the resolver chain today for the other types